docs/intro.md

```diff
@@ -25,7 +17,7 @@ Didn’t we say anything? A few examples of how UOR can be used are:
 - File Management  
 - Hosting static or dynamic web content  
 - Labeling and Dataset management  
-- AI model management - https://asciinema.org/a/wWCR7VY0zC97m3ibd3mKXcsJL  
+- [AI model management](https://asciinema.org/a/wWCR7VY0zC97m3ibd3mKXcsJL)
 - As a relational database  
```

Comment:
Remove references to relational and object relational databases

Reply:
@afflom removing line 21, any other lines need removed?

Reply:
22 as well please